ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -27,7 +27,9 @@
     - x11
     - NTP_TIME
     - gcc_48
+    - gcc_64
```

Comment:
Why gcc 6.4 and not something newer like 7.3+ which contains the Spector backports?

Reply:
Should be ok too, as long as it is 5 or higher. The gcc_64 yml however will install it from source, which takes ages. I think it would be better to install a binary (6.4 or 7.3+)?

Reply:
I think ansible lets you directly download a binary...

Reply:
7.3 is available on a number of releases, but not the older ones.  For example on x86 CentOS69 is used which will need to compile from source.  

I agree its slow, but its a one time setup cost.  I am not sure what the support timelines are for openjdk for supporting gcc7.3+, for OMR the support is there, for OpenJ9 is close from what I've 'heard', its for this reason I suggested 7.3.

Reply:
I've got the PR in to install GCC 7.3 on the build boxes so this shouldn't be needed any more - can you remove from your PR please @johanvos 
Depending on architecture you'll find the gcc in either:
`/opt/rh/devtoolset-2/root/usr/bin/gcc` (LInux/x86-64)
`/usr/local/gcc/bin/gcc-7.3` (Linux/ppc64le & Linux/s390x)